### ISSUE TITLE
Option to remove court

### ISSUE THREAD

**Issue by chrisdesilva** [chrisdesilva/pickup#33]
If a user adds a court, he/she should be able to remove it from the list

**Comment by manAbl** [comment]
I would like to take this issue

**Comment by chrisdesilva** [comment]
Assigned. Thanks!

**Comment by manAbl** [comment]
Can't register to use google maps cause I don't have a credit card :(

**Comment by manAbl** [comment]
Yeah... I can't contribute. Google Maps API keeps asking for a credit card and I don't have one :(

**Comment by chrisdesilva** [comment]
If you're still looking to contribute to the project, check out issue #39
